docs/core/project-sdk/msbuild-props.md

```diff
@@ -304,6 +315,19 @@ In .NET 5 and earlier versions, reference assemblies are always written to the `
 
 For more information, see [Write reference assemblies to intermediate output](../compatibility/sdk/6.0/write-reference-assemblies-to-obj.md).
 
+### PublishRelease
+
+The `PublishRelease` property informs `dotnet publish` to leverage `Release` configuration instead of `Debug` by default. Add it to a `Directory.Build.props` file.
```

Comment:
```suggestion
The `PublishRelease` property informs `dotnet publish` to leverage the `Release` configuration instead of the `Debug` configuration. We recommend adding this property to a `Directory.Build.props` file instead of a project file so that it's evaluated early enough for the configuration change to propagate.
```